gatsby-theme-landing-page/gatsby-config.js

```diff
@@ -2,6 +2,7 @@
 module.exports = (opts = {}) => {
   return {
     plugins: [
+      'gatsby-plugin-image',
```

Comment:
This seems to resolve the error in the logs, but we can adjust this as needed once we get images in Contentful

Reply:
And it looks like the deploy failed due to the same image plugin error we were seeing locally

Reply:
[We need all three: ](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#installation)

```
   `gatsby-plugin-image`,
    `gatsby-plugin-sharp`,
    `gatsby-transformer-sharp`, 
```